project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.8.0
```

Comment:
@ckipp01 will that mean users of this plugin will have to upgrade their sbt to this version?

Reply:
No it doesn't. Since I'm using `pluginCrossBuild / sbtVersion := "1.2.8"` it ensures that it's still compiling and testing against "1.2.8" even though the build itself is managed with 1.8.0.

You can see some more details about this in https://www.scala-sbt.org/1.x/docs/Plugins.html#Creating+an+auto+plugin.